_rst/capo_I-sezione_III-articolo_14-bis.rst

```diff
@@ -0,0 +1,29 @@
+
+  g\) rilascio di pareri tecnici, obbligatori  e  vincolanti,  sugli elementi  essenziali  delle  procedure  di  gara  bandite,  ai  sensi dell'articolo 1, comma 512 della legge 28 dicembre 2015, n.  208,  da Consip  e  dai  soggetti  aggregatori  di  cui  all'articolo  9   del decreto-legge 24 aprile 2014, n. 66,  concernenti  l'acquisizione  di beni  e  servizi  relativi  a  sistemi  informativi  automatizzati  e definiti di carattere strategico nel piano triennale.  Il  parere  è reso entro il termine di quarantacinque giorni dal ricevimento  della relativa richiesta e si  applica  l'articolo  17-bis  della  legge  7 agosto 1990, n.  241,  e  successive  modificazioni.  Ai  fini  della presente lettera per elementi essenziali si intendono l'oggetto della fornitura o del servizio,  il  valore  economico  del  contratto,  la tipologia di procedura  che  si  intende  adottare,  il  criterio  di aggiudicazione e relativa ponderazione, le  principali  clausole  che caratterizzano  le  prestazioni  contrattuali.  Si   applica   quanto previsto nei periodi da 2 a 5 della lettera f); 
+
+  h\) definizione  di  criteri  e  modalità  per  il  monitoraggio sull'esecuzione   dei   contratti   da   parte   dell'amministrazione interessata ((...)); 
+
+  i\) vigilanza sui servizi fiduciari ai sensi dell'articolo 17  del regolamento  UE  910/2014  in  qualità  di  organismo  a  tal   fine designato, sui gestori  di  posta  elettronica  certificata,  sui  (( soggetti di cui all'articolo 34, comma 1-bis, lettera b) )),  nonché sui soggetti, pubblici e privati,  che  partecipano  a  SPID  di  cui all'articolo 64;  nell'esercizio  di  tale  funzione  l'Agenzia  può irrogare per le violazioni accertate a carico dei  soggetti  vigilati le sanzioni amministrative di cui all'articolo  32-bis  in  relazione alla gravità della violazione  accertata  e  all'entità  del  danno provocato all'utenza; 
+
+  l\) ogni altra funzione attribuitale da specifiche disposizioni di legge e dallo Statuto. 
+
+  3\. Fermo restando quanto previsto al  comma  2,  AgID  svolge  ogni altra funzione prevista da leggi  e  regolamenti  già  attribuita  a DigitPA,  all'Agenzia  per  la  diffusione   delle   tecnologie   per l'innovazione ((...)). 
```

Comment:
Weird ((...))

Reply:
That's true, but:

* it's the same in Normattiva
* it's very hard to remove automatically